src/main/scripts/jbang

```diff
@@ -131,6 +131,8 @@ output=$(CLICOLOR_FORCE=1 ${JAVA_EXEC} ${JBANG_JAVA_OPTIONS} -classpath ${jarPat
 err=$?
 if [ $err -eq 0 ]; then
   eval "exec $output"
+elif [ $err -eq 3 ]; then
+  echo "$output"
```

Comment:
okey, now i recall my concern with this one. 

This means that jbang won't be able to have an error *and* print to the console..at least the exit code can't be used to designate what should happen...

should we possibly just say that if err is 0 then we run; but if any other code we print ?

Reply:
But isn't that the situation we have right now anyway?
Currently if you have an error any `echo My Text` commands won't be executed either.

Besides I don't think we want "printing output" to always return non-zero return codes, that way they'd always be considered errors by the shell/caller.  (Perhaps not a huge problem but "not nice").

Btw, before I actually printed the output _also_ in case of any other error codes (and returned that error code, while still returning 0 when the error code was the special one (3)) but I removed that.... and now I can't remember anymore why... 🤔

Reply:
Ah yes, I remember. It's becuase with any other error code you can't know if the output was generated by a command that was trying to print some text to the user or if it was the build/run/??? command trying to generate code to execute.

Perhaps we should designate a range of error codes for "exec" errors and a range for "print" errors? Eg:

 - 0 = OK, execute
 - 1-127 = ERROR (don't print captured stdout)
 - 128 = OK, print, return 0
 - 128-255 = ERROR (but still print captured stdout)

Reply:
Still, we'd also have to think if there's any "rational" use-case for printing captured output in case of errors. Do we have (or foresee) any commands where you could get an error mindway? And if so would it be important to show an unfinished result after the error was already printed?

Reply:
> any "rational" use-case for printing captured output in case of errors. Do we have (or foresee) any commands where you could get an error mindway?

lets say `jbang alias list` prints while it iterates and then errors on accessing one of the catalogs...wouldn't it make sense to still get the ones it could access before it errors ?

Reply:
@quintesse ^